smdebug/tensorflow/utils.py

```diff
@@ -4,17 +4,17 @@
 from enum import Enum
 
 # Third Party
-from tensorflow.contrib.distribute import MirroredStrategy as ContribMirroredStrategy
+import tensorflow.compat.v1 as tf
+from packaging import version
 from tensorflow.python.distribute import values
 
 # First Party
 from smdebug.core.modes import ModeKeys
 
 try:
-    import tensorflow.compat.v1 as tf
+    from tensorflow.contrib.distribute import MirroredStrategy as ContribMirroredStrategy  # TF 1.X
 except ImportError:
-    # For TF 1.13
-    import tensorflow as tf
+    ContribMirroredStrategy = tf.distribute.MirroredStrategy  # TF 2.X
```

Comment:
tf2 doesn't have contrib mirrored strategy remove this line, and remove the usage of this var below

Reply:
line 237

Reply:
use tf.distribute.MirroredStrategy for both? there is a tf.compat.v1.distribute.MirroredStrategy

Reply:
Use tf.distribute.MirroredStrategy for both 1.x and 2.x, but tf.compat.v1.distribute.MirroredStrategy only for 1.x